src/containers/EndingContainer.jsx

```diff
@@ -184,6 +184,20 @@ class LastPage extends React.PureComponent {
 
 export default LastPage;
 
+export const query = graphql`
```

Comment:
the entrypoint for i18next is the top level page containers, any new top level pages will need this page query.